README.md

```diff
@@ -235,3 +235,16 @@ the latest `vault` binary is available in your `PATH`.
 ## Contributing
 
 Feel free to open pull requests with additional features or improvements!
+
+## IAM role auth
```

Comment:
This should probably be place in [Authenticate to Different Auth Backends](https://github.com/ianunruh/hvac#authenticate-to-different-auth-backends)

Reply:
Either that, or that *and* create a new third-level heading under that section.